SystemdMeter.c

```diff
@@ -0,0 +1,188 @@
+#include "SystemdMeter.h"
+
+#include <fcntl.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <unistd.h>
+
+#include "CRT.h"
+#include "StringUtils.h"
+
+static const int SystemdMeter_attributes[] = {
+   SYSTEMD_FAILED,
+   SYSTEMD_NOTICE,
+   SYSTEMD_OK
+};
+
+static char *systemState = NULL;
+static unsigned int nFailedUnits = 0, nInstalledJobs = 0, nNames = 0, nJobs = 0;
```

Comment:
Shouldn't this be `(unsigned int)-1` as used in other places for undefined values?